changelog

```diff
@@ -2,6 +2,7 @@ MSAL Wiki : https://github.com/AzureAD/microsoft-authentication-library-for-andr
 V.NEXT
 ----------
 - [MINOR] MSA UI tests for Brokered Auth (#1856)
+- [MINOR] Daily dev version dependency now uses 1.0.+ (#1886)
```

Comment:
Probably doesn't need a changelog since this doesn't ship to production

Reply:
Yeah... I figured in case partners read this and they were using our daily versions... But probably doesn't make sense for customers to see this. I'll remove it